### ISSUE TITLE
Problem with "tap to start"/videosphere play-on-click component on Safari and iOS 18

### ISSUE THREAD

**Issue by Fishbonecreek** [aframevr/aframe#5603]
**Description:**

I am using A-Frame 1.6.0 (latest) in an 8thwall project here:

https://www.8thwall.com/babeledunnit/grottaazzurra360

( I load A-Frame with <script src="https://aframe.io/releases/1.6.0/aframe.min.js"></script> )

and I have issues on several iPhone models (12, 13 Pro, 16 Pro) with latest iOS (18) AND with Safari. The initial "tap to start" does not work.

It works on Chrome on iOS 18.
It works with Chrome AND Safari on previous iOS (17.6.1, 17.7.1).

so it seems there is a sort of a regression of the play-on-click component on iOS 18 Safari version.

------------

to summarize:

- A-Frame Version:

1.6.0

- Platform / Device:

iPhone 12/13 Pro/16 pro, Safari/Chrome, iOS 17/iOS 18

- Reproducible Code Snippet or URL:

https://www.8thwall.com/babeledunnit/grottaazzurra360

my a-scene:

     <a-assets>
      <video id="video"
               loop
               crossorigin="anonymous"
               playsinline 
               webkit-playsinline>
          <source src='....'>
          <source src='....'>
          Your browser does not support the video tag.
      </video>
      </a-assets>

      <a-videosphere
        rotation="0 -90 0" src="#video"
        play-on-click>
      </a-videosphere>
      <a-camera>
        <a-entity
          position="0 0 -1.5"
          text="align:center;
                width:6;
                wrapCount:100;
                color: white;
                value: Tap/Click to Start"
          hide-on-play="#video">
        </a-entity>
      </a-camera>

**Comment by dmarcos** [comment]
In the A-Frame examples It's not playing because the URL to the video in the example is no longer valid. bitmoving changed the hosting URL. I updated it 

https://github.com/aframevr/aframe/blob/master/examples/boilerplate/360-video/index.html

In your case, you are also using 8th Wall to host. No idea the constraints there. I think the problem is on their side

Try to reproduce with https://glitch.com/~aframe and the video from the example:

https://cdn.bitmovin.com/content/assets/playhouse-vr/progressive.mp4

**Comment by Fishbonecreek** [comment]
thx @dmarcos, 

I was able to pinpoint (but not solve) the problem, which seems to be related to Vimeo. For example, the link

https://player.vimeo.com/progressive_redirect/playback/1010291199/rendition/2160p/file.mp4?loc=external&log_user=0&signature=17366920a4ed29487612b800d280ec802e1f0c96566755f38dde784c93436a49

will work in any browser when opened directly, but will exibit the problem above when served through an A-Frame videosphere in an iOS 18 Safari browser.

I was able to replicate this also on Glitch here:

https://glitch.com/~debugvideosphere

when run on iOS 18, it will be unable to serve my "Grotta Azzurra" 360 video and will fall back to the Bitmovin default video. On iOS 17 everything will work as expected.

This happens with both HLS and "direct links" served by Vimeo.

Obviously this is not an A-Frame problem :)
Thanks.
